lib/hca/rate_limited_search.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module HCA
+  module RateLimitedSearch
+    module_function
+
+    def truncate_ssn(ssn)
+      "#{ssn[0..2]}#{ssn[5..8]}"
+    end
+
+    def combine_traits(user_attributes)
+      user_attributes.to_h.except(:ssn, :middle_name).values.map(&:downcase).join
```

Comment:
I know this data is never leaving our Redis instance on our servers, but should we hash this using `Digest::SHA2` or similar?  Or does it not matter?

Reply:
i can turn it into a hash, it shouldn't hurt